doc/rst/technotes/gpu.rst

```diff
@@ -142,14 +156,16 @@ The following are further requirements for GPU support:
 
 * Specifically for targeting AMD GPUs:
 
-  * ROCm version between 4.x and 5.4 or between ROCm 6.0 and 6.2 must be installed.
+  * ROCm version between 4.x and 5.4 or between ROCm 6.0 and 6.2 must be
+    installed.
 
   * For ROCm 5.x, ``CHPL_LLVM`` must be set to ``system``. Note that, ROCm
     installations come with LLVM. Setting ``CHPL_LLVM=system`` will allow you to
     use that LLVM.
 
-  * For ROCm 6.x, only LLVM 18+ is supported. Currently, only
-    ``CHPL_LLVM=bundled`` is supported due to bugs in LLVM. 
+  * For ROCm 6.x, only ``CHPL_LLVM=bundled`` is supported. The bundled LLVM is
+    version 18 with a patch to support ROCm 6. Said patch is in LLVM 19, as such
+    we expect to support system LLVM 19+ with ROCm 6 in the upcoming releases.
```

Comment:
As far as I know, this is sadly not true. LLVM 19 will likely still have the same bug (although that is untested)

Reply:
:( Then, I'll avoid giving any details here. https://github.com/chapel-lang/chapel/pull/25960/commits/1e8d36657d9232a06b0a3ba96d0844b07a86568a makes it say only:

> For ROCm 6.x, only CHPL_LLVM=bundled is supported.